src/reportSources/legacyFflogs/eventAdapter/translate.ts

```diff
@@ -123,16 +122,18 @@ export class TranslateAdapterStep extends AdapterStep {
 		}
 	}
 
-	private adaptSnapshotEvent(event: DamageEvent | HealEvent): Array<Events['snapshot' | 'actorUpdate']> {
+	private adaptFFLogsEffectEvent(event: DamageEvent | HealEvent): Array<Events['execute' | 'damage' | 'heal' | 'actorUpdate']> {
```

Comment:
`adaptEffectEvent` or `adaptExecuteEvent` methinks. "fflogs" isn't meaningful in a fflogs specific file